### ISSUE TITLE
Case insensitivity

### ISSUE THREAD

**Issue by evanwarner** [rkingon/Craft-Plugin--Redirect-Manager#21]
The ability to make URI matches (regex or basic) case-insensitive would be a very welcome addition.

**Comment by knynkwl** [comment]
Agreed, I kind of feel like this shouldn't be case sensitive at all.

**Comment by BrianTheByrne** [comment]
Yeah this would be extremely helpful to my organization as well.

**Comment by rkingon** [comment]
Have you guys tried using regex and ending it with `/i` -- i think i've done that before

**Comment by JoshStaff** [comment]
@rkingon I can't get this function sensitively-less. Do you have an example of how you got it to work? Looking at the code it looks as though appending /i to the regex input would fail the pregmatch?
